local-tests/tests/wrapped-keys/util.ts

```diff
@@ -1,7 +1,50 @@
 import { LIT_NETWORKS_KEYS } from '@lit-protocol/types';
 import { LIT_CHAINS } from '@lit-protocol/constants';
 import { ethers } from 'ethers';
-import { EthereumLitTransaction } from '@lit-protocol/wrapped-keys';
+import { config } from '@lit-protocol/wrapped-keys';
+import {
+  litActionRepositoryCommon,
+  litActionRepository,
+} from '@lit-protocol/wrapped-keys-lit-actions';
+
+import type {
+  LitActionCodeRepository,
+  LitActionCodeRepositoryCommon,
+  EthereumLitTransaction,
+} from '@lit-protocol/wrapped-keys';
+
+const emptyLitActionRepositoryCommon: LitActionCodeRepositoryCommon = {
+  batchGenerateEncryptedKeys: '',
+};
+
+const emptyLitActionRepository: LitActionCodeRepository = {
+  signTransaction: {
+    evm: '',
+    solana: '',
+  },
+  signMessage: {
+    evm: '',
+    solana: '',
+  },
+  generateEncryptedKey: {
+    evm: '',
+    solana: '',
+  },
+  exportPrivateKey: {
```

Comment:
You're right -- I wanted to move this into the common repository, but didn't want to ship a breaking change here which would mean shipping js-sdk v7 since we don't have independent package versioning yet :(